hathor/wallet/util.py

```diff
@@ -72,6 +71,8 @@ def generate_multisig_address(redeem_script: bytes, version_byte: bytes = settin
         :return: The multisig address
         :rtype: str(base58)
     """
+    settings = get_settings()
+    version_byte = version_byte if version_byte is not None else settings.MULTISIG_VERSION_BYTE
```

Comment:
Same here.

Reply:
Changed in https://github.com/HathorNetwork/hathor-core/pull/770/commits/e2afcd970cff9597b7d5c5528b09e3a971870fc6. I had to change the variable name for the linter

Reply:
Thanks!